test/codec/Codec.spec.ts

```diff
@@ -1499,6 +1504,33 @@ describe('Codec', () => {
         twoWayCheck(SetTimeRequestCodec, message, object);
     });
 
+    itShouldConvert('ListKnownPartiesResponse', ()=>{
```

Comment:
A test for the `PartyDetailsCodec` is missing.

Reply:
Fixed

Reply:
Done